src/foam/nanos/menu/TreeAltView.js

```diff
@@ -15,24 +15,23 @@ foam.CLASS({
   extends: 'foam.u2.view.AltView',
 
   requires: [
-    'foam.nanos.menu.Menu',
-    'foam.u2.view.TableView'
+    'foam.nanos.menu.Menu'
   ],
 
   properties: [
     'relationship'
   ],
 
   methods: [
-    function init(){
-      var relationship = typeof this.relationship === 'object' ? 
+    function init() {
+      var relationship = typeof this.relationship === 'object' ?
         relationship :
         foam.lookup(this.relationship);
 
       this.views = [
         [
-          { 
-            class: 'foam.u2.view.TableView'
+          {
+            class: 'foam.u2.view.ScrollTableView'
```

Comment:
Still better to list it in requires: to help the tooling, even if you don't strictly need it